src/Services/FHIR/FhirPatientService.php

```diff
@@ -69,19 +69,17 @@ public function parsePatientResource($fhirJson)
         $data["postal_code"] = $fhirJson["address"][0]["postalCode"];
         $data["city"] = $fhirJson["address"][0]["city"];
         $data["state"] = $fhirJson["address"][0]["state"];
-        $data["country_code"] = "" ;
+        $data["country_code"] = null;
```

Comment:
any reason for not omitting it and adding it as NULL 😕

Reply:
> any reason for not omitting it and adding it as NULL

Two, first I want to show a null or false will exclude in query build and second, i'm unclear where to add a country code for FHIR international.